core/store/package_manifest.rs

```diff
@@ -83,6 +84,26 @@ impl PackageManifest {
         self.published_at
     }
 
+    pub async fn write(&self, path: &Path) -> Result<(), FromFileError> {
+        let json = serde_json::to_string_pretty(&self).unwrap();
+        fs::write(&path, json)
+            .await
+            .map_err(|err| FromFileError::CouldNotWriteFile {
+                err,
+                file: path.to_path_buf(),
+            })
+    }
+
+    // replaces the existing keys with those of a new manifest
```

Comment:
Let's turn this into an actual documentation comment:
```suggestion
    /// Replaces the existing keys with those of a new manifest.
```